lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -90,6 +90,7 @@ export const TextFieldWithSteppers = forwardRef<
             triggerActionIcon={steppersSectionRight}
             allowDecimals={true}
             value={value && getFormatValue(value, decimals)}
+            noStatusIcon
```

Comment:
```suggestion
            noStatusIcon={noStatusIcon}
```
can we please make it customizable?🙏🏻 
 also, it seems that stories don't react to the prop changes, although it can be fixed later